tasks/run-buildpack-checksums-specs.yml

```diff
@@ -13,6 +13,10 @@ run:
     - |
       set -ex
       cd buildpack-checksums
+      if [ -n "${RUBYGEM_MIRROR}" ]; then
+        gem sources --clear-all --add ${RUBYGEM_MIRROR}
+      fi
+      gem install bundler --no-ri --no-rdoc
       gem install bundle
```

Comment:
Can you remove this `gem install bundle` line as you're now using `gem install bundler`?